colcon_cmake/task/cmake/build.py

```diff
@@ -281,38 +290,53 @@ def _get_msbuild_environment(self, args, env):
             env['CL'] = ' '.join(cl_split)
         return env
 
-    def _get_make_arguments(self, env):
+    def _get_jobs_arguments(self, args, env):
         """
         Get the make arguments to limit the number of simultaneously run jobs.
 
         The arguments are chosen based on the `cpu_count`, e.g. -j4 -l4.
 
+        This only handles some generators, specifically; Makefiles and Ninja
+        derivatives.
```

Comment:
@KazNX nit: let's keep it generic
```suggestion
        This only handles jobs based generators, see `is_jobs_based_generator()`.
```